src/utils/server/pages/props.ts

```diff
@@ -58,19 +59,24 @@ export async function getProjectProps(
 
   try {
     const metadataCid = await getMetadataCidFromContract(projectId)
-
-    const [metadata, isV3ProjectResult] = await Promise.all([
+    const [metadata, hasV2FundingCycle, hasV3FundingCycle] = await Promise.all([
       findProjectMetadata({ metadataCid }),
-      isV3Project(projectId),
+      hasFundingCycle(projectId, CV_V2),
+      hasFundingCycle(projectId, CV_V3),
     ])
 
-    const cv = isV3ProjectResult ? CV_V3 : CV_V2
+    const initialCv = hasV3FundingCycle ? CV_V3 : CV_V2
+
+    const cvs: CV2V3[] = []
+    if (hasV2FundingCycle) cvs.push(CV_V2)
+    if (hasV3FundingCycle) cvs.push(CV_V3)
```

Comment:
We'll use this to determine if we should show the version selector, or just the static badge.

Reply:
we show the selector if there's both v2 and v3 activity ya?

Reply:
:getsit: